hydra-auction-error/src/HydraAuction/Error/Types/AuctionState.hs

```diff
@@ -12,7 +12,7 @@ import HydraAuction.Error.Types.BidTerms (BidTerms'Error (..))
 data Buyer'Error
   = Buyer'Error'EmptyStandingBid
   | Buyer'Error'BuyerVkPkhMismatch
-  | Buyer'Error'InvalidBidTerms [BidTerms'Error]
+  | Buyer'Error'InvalidBidTerms ![BidTerms'Error]
```

Comment:
We have strictness by default in cabal for this.

Reply:
Yes, and apparently it causes PlutusTx plugin errors to have `!` strictness annotations in data types.
https://github.com/input-output-hk/plutus/issues/4193